tests/cypress/integration/4-design-steps/theme-styles-menu.cy.js

```diff
@@ -90,7 +90,7 @@ describe( 'Theme Styles Menu and Preview', function () {
 		} );
 	} );
 
-	it( 'Check for the selected theme in Drawer', () => {
+	it.skip( 'Check for the selected theme in Drawer', () => {
```

Comment:
Why don't we get rid of these from here? The skipped ones in this file? as we have addressed them in the preview step?